ydb/core/tx/schemeshard/ut_restore/ut_restore.cpp

```diff
@@ -4006,6 +4006,61 @@ Y_UNIT_TEST_SUITE(TImportTests) {
             NLs::HasNoRight("+R:bob")
         });
     }
+
```

Comment:
Since you still have to do another commit for the tests to compile. Could you remove the extra newline here?